bundles/binding/org.openhab.binding.comfoair/src/main/java/org/openhab/binding/comfoair/handling/ComfoAirConnector.java

```diff
@@ -423,4 +437,15 @@ private String dumpData(byte[] data) {
         }
         return sb.toString();
     }
+
+    private class CPUWorkaroundThread implements SerialPortEventListener {
+        @Override
+        public void serialEvent(SerialPortEvent event) {
+            try {
+                logger.trace("RXTX library CPU load workaround, sleep forever");
+                Thread.sleep(Long.MAX_VALUE);
+            } catch (InterruptedException e) {
+            }
+        }
+    }
```

Comment:
This is common workaround for the buggy rxtx library. Zwave and eBUS binding uses the same workaround. I've attached the source in zwave binding.

https://github.com/openhab/openhab1-addons/blob/master/bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/ZWaveController.java#L348

https://github.com/openhab/openhab1-addons/blob/master/bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/ZWaveController.java#L1468